firefox-ios/Client/Frontend/Browser/MainMenu/Views/MainMenuViewController.swift

```diff
@@ -318,6 +322,20 @@ class MainMenuViewController: UIViewController,
 
     private func adjustLayout() {
         menuContent.accountHeaderView.adjustLayout()
+        if let screenHeight = view.window?.windowScene?.screen.bounds.height {
+            let isPad = UIDevice.current.userInterfaceIdiom == .pad
```

Comment:
I believe it's better to check the size class instead of checking if the device is an iPad because iPads can have multiple windows, making the device check irrelevant.

Reply:
@PARAIPAN9  is correct. We should check size class here vs iPad, given that we don't know how this is used on iPad.